packages/framework-core/src/booster-graphql-dispatcher.ts

```diff
@@ -155,7 +165,13 @@ export class BoosterGraphQLDispatcher {
     })
     result.errors = result.errors?.map(toGraphQLErrorWithExtensions)
     this.logger.debug('GraphQL result: ', result)
-    return result
+
+    return {
+      data: result.data,
+      errors: result.errors,
+      extensions: result.extensions,
+      responseHeaders: resolverContext.responseHeaders,
+    }
```

Comment:
As the caller already have a reference to `resolverContext`, there's no need to pass it back in the response, we can leave it as it was before:

```suggestion
    return result
```